app/renderer/sweetalert.js

```diff
@@ -0,0 +1,14 @@
+"use strict";
+
+//Methods interface with jQuery
+/* jshint ignore:start */
+module.exports = {
+    toggleModal(title, type, children = {}) {
+        return swal(Object.assign({}, {
```

Comment:
Why do we need an Object.assign(...)?

An idea to improve the wrapper is to add shortcut methods for some common types of modals

And, what are the children?

Reply:
For what I see in other code, children is more like options, variables or parameters. I think the name children is confusing (I would use it for modals within modals or something like that)

Reply:
its a react way of defining children props of a component and that is how i develop/extend components which is actually very common on react. Do you prefer some other naming?

Reply:
You mean the assign?

I'm sorry, I may be a bit lost here, but swal is not a component, is just a method right?

`Object.assign` as you used there is basically creating a copy of the object, this makes sense when the values may be changed (e.g. in a react component) and you don't want them to be changed by the children. This is not the case, as sweetalert is just a method that doesn't change the values.

Making a full copy of the object, effectively, reduces performance and readability, as simply sending title and type as an object should be enough (I may be wrong here, I'm not sure how swal works)


About children naming, basically those are parameters defined by the library like `confirmButtonColor` or `cancelButtonText` right? They are not actually children

Reply:
Yeah, maybe i explained it wrong. Yes swal is not a component but i was using a component like technique to assign properties from the library. Because i am using different kind of properties on different kind of cases which i dont know and i will not know in the future i created a generic function which takes a title, a type and if you want some children properties to extend your configuration (swal properties) you should pass them as object. This will only take effect if swal has some of these properties of course. It will not affect in any case the result from swal.

I think that object extension is readable enough. It can be more readable in es7 with {...object1, ...object2} but again its simple enough. Also performance wise not sure why you are mentioning this because i dont think that this has some real impact on the application. 

Do you have another idea on how to extend the object with more options for now and for future use? Let me know your opinion in that.

Thanks in advance

Reply:
Oh sorry, I misread the method Object assign, I though it wasn't being used on children.
You are right, that's the way to go. :+1: 

So, my only comments now are the use of the name "children" and a minor code improvement is that I think you don't need that first argument in assign (you can directly use `Object.assign({title...},children);`